spec.html

```diff
@@ -227,11 +229,14 @@ <h1>ResizableArrayBuffer.prototype [ @@toStringTag ]</h1>
   <h1>Modifications to Abstract Operations for SharedArrayBuffer Objects</h1>
 
   <emu-clause id="sec-allocatesharedarraybuffer" aoid="AllocateSharedArrayBuffer">
-    <h1>AllocateSharedArrayBuffer ( _constructor_, _byteLength_<ins>[ , _maxByteLength_ ]</ins> )</h1>
-    <p>The abstract operation AllocateSharedArrayBuffer takes arguments _constructor_<del> and</del><ins>,</ins> _byteLength_<ins>, and _maxByteLength_</ins>. It is used to create a SharedArrayBuffer object. It performs the following steps when called:</p>
+    <h1>AllocateSharedArrayBuffer ( _constructor_, _byteLength_<ins>[ , _requestedMaxByteLength_ ]</ins> )</h1>
+    <p>The abstract operation AllocateSharedArrayBuffer takes arguments _constructor_<del> and</del><ins>,</ins> _byteLength_<ins>, and _requestedMaxByteLength_</ins>. It is used to create a SharedArrayBuffer object. It performs the following steps when called:</p>
     <emu-alg>
       1. <ins>Let _slots_ be &laquo; [[ArrayBufferData]] &raquo;.</ins>
-      1. <ins>If _maxByteLength_ is present, append [[ArrayBufferByteLengthData]] and [[ArrayBufferMaxByteLength]] to _slots_.</ins>
+      1. <ins>If _requestedMaxByteLength_ is present, then:</ins>
+        1. <ins>Append [[ArrayBufferByteLengthData]] and [[ArrayBufferMaxByteLength]] to _slots_.</ins>
+        1. <ins>Let _maxByteLength_ be an implementation-defined value that is &ge; _requestedMaxByteLength_ and a power of 2.</ins>
+        1. <ins>NOTE: Implementations may choose to round up the requested maximum byte length, for example, to be on a page boundary.</ins>
       1. <ins>Else, append [[ArrayBufferByteLength]] to _slots_.</ins>
```

Comment:
(just experimenting with GitHub UI, maybe I need to click the "start a review" button so that the code review UI would appear?)